section_12/hello_from_decorator.py

```diff
@@ -0,0 +1,13 @@
+from curses import wrapper
+from functools import wraps
+def hello_from_decorator(function):
+    @wraps(function)
+    def wrapper(*args, **kwargs):
+        result = function(*args, **kwargs)
+        return str(result) + " Hello from decorator!"
+    return wrapper()
+
+@hello_from_decorator
+def sum_with_gen():
+    return sum(number for number in range(10000000))
+print(sum_with_gen)
```

Comment:
Rebuild this, so `sum_with_gen` returns the value it produces, and the decorator prints its own value separately